src/compile.js

```diff
@@ -525,6 +525,15 @@ Compiler.prototype.ccall = function (e) {
     }
     else {
         out ("$ret;"); // This forces a failure if $ret isn't defined
+        if (Sk.python3 && e.func.id && e.func.id.v === "super" && args.length == 0) {
```

Comment:
It's not the same as cptyhon, cpyhton get the first variable of the stack frame when calling `super`. But we don't have stackframes, and javascript doesn't allow us to inspect the scope. I saw no other obvious way of solving this. The only caveat with this solution is that the first variable of the function needs to be called `self`. I did this because I have no way of determining if the `super` function is called from a bound or unbound function.